virtcontainers/kata_shim_test.go

```diff
@@ -272,6 +281,54 @@ func TestKataShimStartWithConsoleNonExistingFailure(t *testing.T) {
 	testKataShimStart(t, sandbox, params, true)
 }
 
+func ioctl(fd uintptr, flag, data uintptr) error {
```

Comment:
@GabyCT I guess these were part of hyperstart_shim_test? Even though looks like these are being used by tests, I prefer to move these into existing `utils` file/package.

Reply:
It looks like there was already an oictl function in utils_linux, I modified it a little to include this use case. Thank you :)